src/main/java/teammates/sqllogic/core/CoursesLogic.java

```diff
@@ -60,6 +69,72 @@ public Course getCourse(String courseId) {
         return coursesDb.getCourse(courseId);
     }
 
+    /**
+     * Returns a list of {@link Course} for all courses a given student is enrolled in.
+     *
+     * @param googleId The Google ID of the student
+     */
+    public List<Course> getCoursesForStudentAccount(String googleId) {
+        List<Student> students = usersLogic.getAllStudentsByGoogleId(googleId);
+
+        List<String> courseIds = students.stream()
+                .filter(student -> !getCourse(student.getCourseId()).isCourseDeleted())
+                .map(student -> student.getCourseId())
+                .collect(Collectors.toList());
+
+        return coursesDb.getCourses(courseIds);
+    }
+
+    /**
+     * Returns a list of {@link CourseAttributes} for all courses for a given list of instructors
+     * except for courses in Recycle Bin.
+     */
+    public List<Course> getCoursesForInstructors(List<Instructor> instructorsList) {
```

Comment:
```suggestion
    public List<Course> getCoursesForInstructors(List<Instructor> instructors) {
```

Really don't need the word list, that's what the type is for.